framework/include/actions/MaterialDerivativeTestAction.h

```diff
@@ -47,5 +47,5 @@ class MaterialDerivativeTestAction : public Action, public DerivativeMaterialPro
   const bool _second;
 
   /// every derivative given by a list of variables to derive w.r.t
-  std::map<MaterialPropertyName, std::vector<VariableName>> _derivatives;
+  std::map<MaterialPropertyName, std::vector<SymbolName>> _derivatives;
```

Comment:
update the comment? It is not limited to `variables` any more, correct?

Reply:
In this action we only derive w.r.t. MOOSE variables